src/libs/Notification/PushNotification/index.native.js

```diff
@@ -21,25 +22,25 @@ function pushNotificationEventCallback(eventType, notification) {
         payload = JSON.parse(payload);
     }
 
-    console.debug(`[PUSH_NOTIFICATION] ${eventType}`, {
+    Log.info(`[PUSH_NOTIFICATION] ${eventType}`, false, {
         title: notification.title,
         message: notification.alert,
         payload,
     });
```

Comment:
I don't think we should be logging this stuff. It could be anything? Could be user chats which should not be logged?

Reply:
Now just logging that the callback was executed for a given event type